packages/dynamodb-data-mapper/src/namedParameters/CreateTableOptions.ts

```diff
@@ -4,8 +4,11 @@ import { PerIndexOptions } from './SecondaryIndexOptions';
 export interface CreateTableOptions extends ProvisionedThroughput {
     streamViewType?: StreamViewType;
     indexOptions?: PerIndexOptions;
+    billingMode?: BillingMode;
```

Comment:
Could this be made conditionally required, i.e., `billingMode` must be `'PAY_PER_REQUEST'` if no read or write capacity units are specified? I think `CreateTableOptions` would need to be redeclared as a union of what it currently is or the proposed change.
```suggestion
interface BaseCreateTableOptions {
    streamViewType?: StreamViewType;
    indexOptions?: PerIndexOptions;
    billingMode?: BillingMode;
}

export interface ProvisionedCreateTableOptions extends ProvisionedThroughput, BaseCreateTableOptions {
    billingMode?: 'PROVISIONED';
}

export interface OnDemandCreateTableOptions extends BaseCreateTableOptions {
    billingMode: 'PAY_PER_REQUEST';
}

export type CreateTableOptions = ProvisionedCreateTableOptions | OnDemandCreateTableOptions;
```